src/screens/gm/gameMasterScreen.cpp

```diff
@@ -444,14 +444,32 @@ void GameMasterScreen::onMouseUp(sf::Vector2f position)
         break;
     case CD_BoxSelect:
         {
+            bool shift_down = InputHandler::keyboardIsDown(sf::Keyboard::LShift) || InputHandler::keyboardIsDown(sf::Keyboard::RShift);
+            bool ctrl_down = sf::Keyboard::isKeyPressed(sf::Keyboard::LControl) || sf::Keyboard::isKeyPressed(sf::Keyboard::RControl);
+            bool alt_down = InputHandler::keyboardIsDown(sf::Keyboard::LAlt) || InputHandler::keyboardIsDown(sf::Keyboard::RAlt);
```

Comment:
@daid Control key strokes seem not to trigger a `sf::Event::KeyEvent` on their own. I did not dare adapt the `InputHandler::keyboardIsDown` logic because I have no idea what the side effects could be.

Reply:
Using sf::keyboard is fine in this case. Would be nice to place a comment in the code why that is done, instead of only on the pull request. That not someone else comes by and tries to "fix" the inconsistency.

SFMLs input handling isn't super great. So I'm not surprised.